src/libs/actions/App.js

```diff
@@ -48,8 +50,8 @@ function setLocale(locale) {
  * @param {string} url relative URL starting with `/` to open in expensify.com
  */
 function openSignedInLink(url = '') {
-    API.GetShortLivedAuthToken().then((response) => {
-        Linking.openURL(`${CONFIG.EXPENSIFY.URL_EXPENSIFY_COM}${url}${url.indexOf('?') === -1 ? '?' : '&'}authToken=${response.authToken}&email=${encodeURIComponent(response.email)}`);
+    API.GetShortLivedAuthToken().then(({shortLivedAuthToken}) => {
+        Linking.openURL(`${CONFIG.EXPENSIFY.URL_EXPENSIFY_COM}${url}${url.indexOf('?') === -1 ? '?' : '&'}authToken=${shortLivedAuthToken}&email=${currentUserEmail}`);
```

Comment:
We don't need to `encodeURIComponent` the email anymore?

Reply:
Why not? I thought a `+` in a url mean that it's a `SPACE` and would mess up query params?

Reply:
With the encodeURIComponent, it doesn't log me in for an email like `joe+test@expensifail.com`, with the current code the `+` emails work for me.

Reply:
Sorry this was my bad I moved it back so hopefully less confusing now.